src/main/java/uk/gov/companieshouse/insolvency/delta/service/api/ApiClientServiceImpl.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.companieshouse.insolvency.delta.service.api;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.api.http.ApiKeyHttpClient;
+import uk.gov.companieshouse.api.http.HttpClient;
+import uk.gov.companieshouse.api.insolvency.InternalCompanyInsolvency;
+import uk.gov.companieshouse.api.model.ApiResponse;
+import uk.gov.companieshouse.logging.Logger;
+
+/**
+ * Service that sends REST requests via private SDK.
+ */
+@Primary
+@Service
+public class ApiClientServiceImpl extends BaseApiClientServiceImpl implements ApiClientService {
+
+    @Value("${api.internal-api-key}")
+    private String chsApiKey;
+
+    @Value("${api.api-url}")
+    private String apiUrl;
+
+    @Value("${api.payments-url}")
```

Comment:
question: Do we need this at all? Not quite sure there's any payment integrations on this component. Same goes for the related env var

Reply:
Thanks for removing